src/components/ComponentFilter/index.jsx

```diff
@@ -1,21 +1,13 @@
 /*
  * @copyright   Copyright (C) 2022 AesirX. All rights reserved.
  * @license     GNU General Public License version 3, see LICENSE.
-*/
+ */
```

Comment:
Small space, can you please fix so the headers are unified? (it happened in other modified files too)

Reply:
The extension Prettier  - Code Formatter did it on its own